recipe/build.sh

```diff
@@ -1,19 +1,17 @@
 #!/bin/bash
 
-# default is 10.5 for some reason, which doesn't work
+EXTRA_CMAKE_ARGS=""
 if [[ `uname` == 'Darwin' ]];
 then
-    export MACOSX_DEPLOYMENT_TARGET=10.7
-    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+    export CMAKE_OSX_DEPLOYMENT_TARGET=""
```

Comment:
Sure it should be 10.7 either actually. This is a holdover from a long time ago. Just suggesting we remove both.